packages/components/src/components/hds/dropdown/toggle/icon.ts

```diff
@@ -76,7 +92,7 @@ export default class HdsDropdownToggleIconComponent extends Component {
    * @default 24
    * @description ensures that the correct icon size is used
    */
-  get iconSize() {
+  get iconSize(): '24' | '16' {
```

Comment:
should we import the sizes from the icon?
```suggestion
  get iconSize(): FlightIconSignature['Args']['size'] {
```

Reply:
ah, we have sizes now; there's another instance in our codebase where we have them as values, I'll make sure to update that too